src/NavConfig.jsonc

```diff
@@ -1,32 +1,47 @@
-{
-  "navMenu": {
-    "groups": [
-      {
-        "identity": "main",
-        "tools": [
-          {
-            "identity": "DevHome.Dashboard",
-            "assembly": "DevHome.Dashboard",
-            "viewFullName": "DevHome.Dashboard.Views.DashboardView",
-            "viewModelFullName": "DevHome.Dashboard.ViewModels.DashboardViewModel",
-            "icon": "f246"
-          },
-          {
-            "identity": "DevHome.SetupFlow",
-            "assembly": "DevHome.SetupFlow",
-            "viewFullName": "DevHome.SetupFlow.Views.SetupFlowPage",
-            "viewModelFullName": "DevHome.SetupFlow.ViewModels.SetupFlowViewModel",
-            "icon": "f156"
-          },
-          {
-            "identity": "DevHome.ExtensionLibrary",
-            "assembly": "DevHome.ExtensionLibrary",
-            "viewFullName": "DevHome.ExtensionLibrary.Views.ExtensionLibraryView",
-            "viewModelFullName": "DevHome.ExtensionLibrary.ViewModels.ExtensionLibraryViewModel",
-            "icon": "ea86"
-          }
-        ]
-      }
-    ]
-  }
-}
\ No newline at end of file
+{
```

Comment:
I think you may have changed the line endings on this file, can you put them back to crlf?

Reply:
should be ok now?

Reply:
Was still LF. I pushed a commit with just this change, I hope you don't mind.

Reply:
![ThankYesGIF](https://github.com/microsoft/devhome/assets/22989529/fde0b229-fbe0-45b7-81d3-cfb9e2c6843b)